assets/components/organisms/contact/contact-banner.twig

```diff
@@ -1,32 +1,30 @@
-<div class="bg-white p-4 p-md-5">
-  <div class="row">
-    <div class="col-md-6">
-      <h3>Contact</h3>
-      <p>Les guichets des « services aux étudiant-e-s » est le point de contact central pour traiter vos demandes.</p>
-      <div class="card card-body card-sm mb-2 flex-row flex-wrap justify-content-between justify-content-sm-start">
-        <div class="mr-3 w-sm-50">Lundi à jeudi</div>
-        <div><strong><time datetime="09:00">09:00</time> <span aria-label="à">›</span> <time datetime="18:00">18:00</time></strong></div>
-      </div>
-      <div class="card card-body card-sm mb-4 flex-row flex-wrap justify-content-between justify-content-sm-start">
-        <div class="mr-3 w-sm-50">Vendredi</div>
-        <div><strong><time datetime="09:30">09:30</time> <span aria-label="à">›</span> <time datetime="17:00">17:00</time></strong></div>
-      </div>
-      <p>
-        <a class="link-pretty" href="mailto:services.etudiants@epfl.ch">services.etudiants@epfl.ch</a>
-      </p>
-      <hr>
-      <p>
-        +41 21 693 43 45
-      </p>
-      <hr>
-      <address>
-        EPFL AA-DAF<br>
-        BP 1229 (bâtiment BP) Station 16<br>
-        CH - 1015 Lausanne
-      </address>
+<div class="row">
+  <div class="col-md-6">
+    <h3>Contact</h3>
+    <p>Les guichets des « services aux étudiant-e-s » est le point de contact central pour traiter vos demandes.</p>
+    <div class="card card-body card-sm mb-2 flex-row flex-wrap justify-content-between justify-content-sm-start">
+      <div class="mr-3 w-sm-50">Lundi à jeudi</div>
+      <div><strong><time datetime="09:00">09:00</time> <span aria-label="à">›</span> <time datetime="18:00">18:00</time></strong></div>
     </div>
-    <div class="col-md-6 d-flex flex-column">
-      {% include '@molecules/map/map.twig' %}
+    <div class="card card-body card-sm mb-4 flex-row flex-wrap justify-content-between justify-content-sm-start">
+      <div class="mr-3 w-sm-50">Vendredi</div>
+      <div><strong><time datetime="09:30">09:30</time> <span aria-label="à">›</span> <time datetime="17:00">17:00</time></strong></div>
     </div>
+    <p>
+      <a class="link-pretty" href="mailto:services.etudiants@epfl.ch">services.etudiants@epfl.ch</a>
```

Comment:
`link-pretty` are not very pretty on a grey background... You must choose between a normal link or adding a contextual variant style for grey background.

<img width="282" alt="capture d ecran 2018-08-24 a 09 13 08" src="https://user-images.githubusercontent.com/2025924/44570385-ef291e00-a77d-11e8-9d16-68c6c4012971.png">

Reply:
yep done 83c31f1